js/views/composer.js

```diff
@@ -187,6 +208,22 @@ define(function(require) {
 				_this.saveDraft();
 			}, this.draftInterval);
 		},
+		toggleEditor: function()
```

Comment:
@Gomez and I tested your latest commit and we couldn't see that button on the interface.